<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="screen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>701</width>
      <height>521</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>사진,영상표시 위치</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCamera">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>70</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecord">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSign">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>590</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="shapeB">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="shapeR">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="shapeG">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>line color Edit</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>590</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="binary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Binary</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="blur">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Blur</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="filtering">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Filtering</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cannyedge">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Canny edge</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="threshold">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnHSV">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>340</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>HSV Edit</string>
    </property>
   </widget>
   <widget class="QSlider" name="sslider">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>400</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hslider_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>400</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRed">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>89</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGreen">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>89</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlue">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>230</y>
      <width>89</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
